eventsourcing-jakarta/pom.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing the [copyright notice](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml) in this file.

Reply:
Added